documentation/asciidoc/computers/raspberry-pi/bcm2711-bootloader.adoc

```diff
@@ -480,3 +547,52 @@ Default: `1`
 After reading `config.txt` the GPU firmware `start4.elf` reads the bootloader EEPROM config and checks for a section called `[config.txt]`. If the `[config.txt]` section exists then the contents from the start of this section to the end of the file is appended in memory, to the contents of the `config.txt` file read from the boot partition.  This can be used to automatically apply settings to every operating system, for example, dtoverlays.
 
 WARNING: If an invalid configuration which causes boot to fail is specified then the bootloader EEPROM will have to be re-flashed.
+
+[[bootloader_update_stable]]
+=== Updating to the LATEST / STABLE bootloader
+
+The DEFAULT version of the bootloader is only updated for CRITICAL fixes and major releases. The LATEST / STABLE bootloader is updated more often to include the latest fixes and improvements.
+
+Advanced users can switch to the LATEST / STABLE bootloader to get the latest functionality.
+Open a command prompt and start `rasp-config`.
+
+----
+sudo raspi-config
+----
+
+Navigate to `Advanced Options` and then `Bootloader Version`. Select `Latest` and choose `Yes` to confirm. Select `Finish` and confirm you want to reboot. After the reboot, open a command prompt again and update your system.
+
+----
+sudo apt-get update
+sudo apt-get install rpi-eeprom
+rpi-eeprom-update
+----
+
+You should see that a more recent version of the bootloader is available and that you're using the `stable` release.
+
+----
+*** UPDATE AVAILABLE ***
+BOOTLOADER: update available
+   CURRENT: Tue 25 Jan 14:30:41 UTC 2022 (1643121041)
+    LATEST: Thu 10 Mar 11:57:12 UTC 2022 (1646913432)
+   RELEASE: stable (/lib/firmware/raspberrypi/bootloader/stable)
+            Use raspi-config to change the release.
+----
+
+Now you can update your bootloader.
+
+----
+sudo rpi-eeprom-update -a
+sudo reboot
+rpi-eeprom-update
+----
```

Comment:
I think having a command listed _after_ `sudo reboot` in the same code-block is a little odd. You should probably have `sudo reboot` as the last line, and then say something like "If you run `rpi-eeprom` again after your Raspberry Pi has rebooted, you should now see..."